CHANGELOG.md

```diff
@@ -36,6 +39,11 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Change blogpost detail template to display author even in published mode
   except if its placeholder is empty.
 
+### Fixed
+
+- Hide title _To be scheduled_, _Upcoming_, _Ongoing_ and _Archived_ when all
+  course runs have catalog visibility `hidden`.
+
```

Comment:
Can you split this in two commits, please ?

Reply:
Done, splited.

I didn't found a style for a button with a grey background, so I've used the normal button style, like this:
![image](https://user-images.githubusercontent.com/67018/160160241-661bb2bb-5e61-4704-8a2e-1e9cdde0570d.png)

What do you think?

Reply:
It's too heavy visually 😕

I tried to find something lighter and I suggest that :
<img width="316" alt="image" src="https://user-images.githubusercontent.com/9265241/160811349-c481c523-7bc7-4189-94a1-cfc17ada5b5d.png">

I push you a commit with my suggestion if you want to test it.

@sampaccoud What do you think?

Reply:
I think it's very important that registration remains the biggest call to action. I had to sleep on it but now it seems clear that the "contact us" button musts go away... to the top menu for example as we did on fun-mooc.fr...
Then the registration should be a red button with the same "subscribe now" CTA as when there is only one course... with the sentence above the button "2 course runs are currently open for this course". 
WDYT?

Reply:
Like this?
![image](https://user-images.githubusercontent.com/67018/161036177-52790e18-c7f9-4c33-9c22-c83d634d24d7.png)

I've updated the PR to have this style.

Reply:
The CTA should be "Enroll now" imo

Reply:
or "choose now"?

Reply:
Personally, I prefer to remove the eye and use a CTA button with the text "Choose now". 
Don't forget that it will scroll the browser to the open course runs.

Reply:
+1 for "choose now" and no eye icon. :+1:

Reply:
So @igobranco removes the eye icon (and all stuff which relies on) and rename the button into `Choose now` then lets use `.button .button--primary` classes on it to have a big red button.

After that, I think we will be able to merge this PR !

Reply:
Done.